ts/Accessibility/Accessibility.ts

```diff
@@ -354,6 +354,7 @@ namespace Accessibility {
     export declare class ChartComposition extends Chart {
         options: Required<Options>;
         series: Array<SeriesComposition>;
+        wasAccessibilityEnabled?: boolean;
```

Comment:
Wouldn't it be better to have that kind of property placed in the `chart.accessibility` object instead of `chart`?

Reply:
Yes, you are right! Will fix it!